fluss-client/src/test/java/com/alibaba/fluss/client/security/acl/FlussAuthorizationITCase.java

```diff
@@ -190,6 +197,14 @@ void testNoAuthorizer() throws Exception {
                                             .get();
                                 })
                         .hasMessageContaining("No Authorizer is configured.");
+
+                // test initWriter without authorizer and empty table paths
+                FlussConnection flussConnection = (FlussConnection) connection;
+                TabletServerGateway tabletServerGateway =
+                        flussConnection.getMetadataUpdater().newTabletServerClientForNode(0);
+                InitWriterResponse response =
+                        tabletServerGateway.initWriter(new InitWriterRequest()).get();
+                assertThat(response.getWriterId()).isEqualTo(0);
```

Comment:
should use `isGreaterThanOrEqualTo`, because the zk id counter maybe increased by other tests.